geode-redis/src/distributedTest/java/org/apache/geode/redis/executors/ExpireDUnitTest.java

```diff
@@ -115,7 +113,7 @@ public static void tearDown() {
   public void expireOnOneServer_shouldPropagateToAllServers() {
     String key = "key";
 
-    jedis1.set(key, "value");
+    jedis1.sadd(key, "value");
```

Comment:
Is there a reason for switching from string values to Sets?

Reply:
c

Reply:
early on this branch, the new expiration was only working on sets and hashes. So I changed these tests to use sets instead of strings. It seems like this test is okay with any data type so I left it using a set.